artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/server/impl/RemotingServiceImpl.java

```diff
@@ -514,8 +514,11 @@ public void connectionCreated(final ActiveMQComponent component,
       }
 
       ConnectionEntry entry = protocol.createConnectionEntry((Acceptor) component, connection);
-      server.callBrokerPlugins(server.hasBrokerPlugins() ? plugin -> plugin.afterCreateConnection(entry.connection) : null);
-
+      try {
+         server.callBrokerPlugins(server.hasBrokerPlugins() ? plugin -> plugin.afterCreateConnection(entry.connection) : null);
+      } catch (Exception e) {
+         throw new IllegalStateException("Error executing afterCreateConnection plugin method: " + e.getMessage(), e);
```

Comment:
shouldn't you just log on this case...

I would expect any failures on the broker plugin to just be logged.